integration-test/src/main/java/org/apache/iotdb/it/env/cluster/env/AbstractEnv.java

```diff
@@ -242,7 +243,30 @@ public String getTestClassName() {
     return "UNKNOWN-IT";
   }
 
-  public void testWorking() {
+  private Map<String, Integer> countNodeStatus(Map<Integer, String> nodeStatus) {
+    Map<String, Integer> result = new HashMap<>();
+    nodeStatus.values().forEach(status -> result.put(status, result.getOrDefault(status, 0) + 1));
+    return result;
+  }
+
+  private final Predicate<Map<Integer, String>> statusCheckNoUnknown =
+      nodeStatus -> countNodeStatus(nodeStatus).getOrDefault("Unknown", 0) == 0;
+
+  private final Predicate<Map<Integer, String>> statusCheckOneUnknownThreeRunning =
+      nodeStatus -> {
+        Map<String, Integer> count = countNodeStatus(nodeStatus);
+        return count.getOrDefault("Unknown", 0) == 1 && count.getOrDefault("Running", 0) == 3;
+      };
+
+  public void testWorkingAllFine() {
+    testWorking(statusCheckNoUnknown);
+  }
```

Comment:
It's better to implement a `statusCheckAllRunning` here.
```suggestion
  public void testWorkingAllFine() {
    testWorking(statusCheckAllRunning);
  }
```

Because the NodeStatus might be extended in the future. And we may test other status like `ReadOnly`

Reply:
Reasonable. 
Besides, considering these two Predicate won't be used elsewhere, I inlined them.